blocks/feed/feed.js

```diff
@@ -48,7 +48,16 @@ const resultParsers = {
     return blockContents;
   },
 
-  highlight: (results, blockCfg) => {
+  highlight: async (results, blockCfg, block = '') => {
```

Comment:
Let's see if we can use the columns block here for the use case this change addresses sine it seems more like a columns block instead of a highlight block. Can be taken up separately from this pr.

Reply:
Sure @sdmcraft  . Point noted.